src/main/java/appeng/recipes/handlers/Inscribe.java

```diff
@@ -63,7 +64,12 @@ public void register() throws RegistrationError, MissingIngredientError
 		final ItemStack output = this.getOutput().getItemStack();
 		final InscriberProcessType type = InscriberProcessType.INSCRIBE;
 
-		final IInscriberRecipe recipe = new InscriberRecipe( inputs, output, top, bot, type );
+		final IInscriberRecipe recipe = builder.withInputs( inputs )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Parameter 1 to this call is marked "javax.annotation.Nonnull" but null is passed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2637)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Parameter 1 to this call is marked "javax.annotation.Nonnull" but null is passed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2637)